Docs-css/Docs-layout.css

```diff
@@ -10,11 +10,17 @@
   border-radius: 8px;
 }
 
+.code-container{
+  width: 96%;
+  margin: 2rem 0 !important; 
```

Comment:
we should avoid it !important

Reply:
Yes totally agree but that was the only option to overwrite prism.css class